src/Lecture4.hs

```diff
@@ -226,7 +318,10 @@ implement the next task.
 -}
 
 combineRows :: NonEmpty Row -> Stats
-combineRows = error "TODO"
+combineRows = combine . fmap rowToStats
+  where
+    combine :: NonEmpty Stats -> Stats
+    combine (x :| xs) = foldl' (<>!) x xs
```

Comment:
Just `<>` would be enough here because `foldl'` already does the same thing as `<>!` 
```suggestion
    combine (x :| xs) = foldl' (<>) x xs
```